### ISSUE TITLE
rename .bin back to .time

### ISSUE THREAD

**Issue by tj** [tj/d3-bar#3]
not sure what I was thinking, bin makes no sense here haha (it does for my use-case but meh)

**Comment by huytd** [comment]
How about `key` and `value`? it's more general for data, right?

**Comment by tj** [comment]
yep, just need to migrate my internal stuff first. It only supports time for now but I suppose that could change
